### DISCUSSION TITLE
Is it normal converging after 3 epochs?

### DISCUSSION THREAD

**Discussion by hforghani** [speechbrain/speechbrain#2862] [Q&A]
I was finetuning Whisper-large-v3 on a dataset containing ~400,000 Persian samples. I wanted to emphasize some subset of the dataset, hence I repeated those. Finally the dataset reached to ~1,200,000 samples. In the plot below you observe training loss, validation loss, CER, and WER while finetuning. Plots start from epoch 243 since already finetuned on another dataset up to epoch 243.

![plot_metrics__recipes_VBoxMultiple_ASR_transformer_results_train_whisper_1986_fa_train_logtxt](https://github.com/user-attachments/assets/95c5ca48-30fe-49ac-a246-996590c48590)

Why WER and CER converged after 3 epochs? Is it normal?

@Adel-Moumen @TParcollet
